addon/-private/sprite.js

```diff
@@ -435,7 +435,7 @@ export default class Sprite {
     }
     this._inInitialPosition = true;
     if (this._offsetSprite) {
-      this._initialBounds = relativeBounds(this.element.getBoundingClientRect(), this._offsetSprite.initialBounds);
+      this._initialBounds = relativeBounds(this._getScaledBoundingClientRect(), this._offsetSprite.initialBounds);
     } else {
       this._initialBounds = this.element.getBoundingClientRect();
```

Comment:
I think this branch of the `if` needs to follow the same coordinate system as the other, so your change would need to go here too.